docs/testing/reliability-testing.md

```diff
@@ -205,17 +208,21 @@ On top of the previous scenarios, we support running ad-hoc load tests. They can
 
 It is as easy as it sounds; we can label an existing PR with the [**benchmark**](https://github.com/camunda/camunda/labels/benchmark) label, which triggers a [GitHub Workflow](https://github.com/camunda/camunda/blob/main/.github/workflows/zeebe-pr-benchmark.yaml). The workflow will build a new Docker image, based on the PR branch, and deploy a new load test against this version.
 
-This method allows no specific configuration or adjustment. If this is needed, triggering the [Zeebe Benchmark GitHub Workflow](https://github.com/camunda/camunda/actions/workflows/zeebe-benchmark.yml) is recommended.
+This method allows no specific configuration or adjustment. If this is needed, triggering the [Camunda load test GitHub workflow](https://github.com/camunda/camunda/actions/workflows/camunda-load-test.yml) is recommended.
 
-##### Trigger Zeebe Benchmark GitHub Workflow
+##### Trigger Camunda load test GitHub Workflow
 
-The [Zeebe Benchmark GitHub Workflow](https://github.com/camunda/camunda/actions/workflows/zeebe-benchmark.yml) is the easiest way to run a load test for a specific branch or main (default) with more customization.
+The [Camunda load test GitHub workflow](https://github.com/camunda/camunda/actions/workflows/camunda-load-test.yml) is the **easiest** way to run a load test for a specific branch or main (default) with more customization. We support to set up load tests for different Camunda/Zeebe versions, by selecting the specific workflow revision as part of the workflow dispatch form (UI).
 
-Existing Docker images can be reused if needed, which is especially useful if we want to recreate tests (like for weekly load tests).
+It allows high customization:
 
-Arbitrary Helm arguments can be passed, making it easier to customize the load tests.
+* Specification of the Camunda/Zeebe version to test against (by selecting the workflow revision) - will make sure to use the right Camunda Platform Helm Chart version and values file.
+* Specification of the branch to test against (default: main) - will build a Docker image based on the specified branch.
```

Comment:
🔧 we could also specify that branch is used for the load tests applications as well